llm-servers/deepsparse/README.md

```diff
@@ -0,0 +1,67 @@
+# DeepSparse Deployment
+
+This deployment can be used to serve optimized models for running on CPU, such as [these]((https://sparsezoo.neuralmagic.com/?modelSet=generative_ai&tasks=text_generation)).  
+By default, it's deploying an optimized [granite-7b](https://huggingface.co/nm-testing/granite-7b-lab-pruned50-quant-ds) instruct model.
+
+A `test_request.ipynb` example is present in the folder to test a connection to the Service once deployed.
+
+## Installation
+
+The default installation deploys an optimized [granite-7b](https://huggingface.co/nm-testing/granite-7b-lab-pruned50-quant-ds) instruct model. See [Advanced installation](#advanced-installation) for instructions on how to change the model as well as various settings.
+
+### Automated Deployment
+
+- Use the OpenShift GitOps/ArgoCD Application definition at `gitops/deepsparse-app.yaml`
+
+### Manual Deployment using Kustomize
+
+After logging in to OpenShift, run the following commands:
+
+```bash
+$ oc new-project deepsparse-text-generation
+$ kustomize build https://github.com/rh-aiservices-bu/llm-on-openshift.git/llm-servers/deepsparse/gitops | oc apply -f -
```

Comment:
don't  you need to skip deepsparse-app.yaml in this case?

Reply:
You mean to make it clearer that it's either deepsparse-app.yaml or kustomize?

Reply:
yep

Reply:
Added some text to hopefully clarify that there are multiple different deployment options